lib/workers/repository/process/lookup/index.ts

```diff
@@ -334,7 +334,7 @@ export async function lookupUpdates(
         if (!res.updates.some((update) => update.updateType === 'pin')) {
           // pin digest
           res.updates.push({
-            updateType: 'pin',
+            updateType: 'pinDigest',
```

Comment:
what's the difference between `pin` and `pinDigest` ? looks something mixed to me.

Reply:
I also forget this logic. Maybe we decided not to pin version *and* digest at the same time and to pin version first?